locales/zh_CN/LC_MESSAGES/vector.po

```diff
@@ -3,178 +3,162 @@
 # This file is distributed under the same license as the taichi package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: taichi 0.5.14\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-05-12 14:33+0800\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2020-05-18 22:29+0800\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../vector.rst:4
 msgid "Vectors"
-msgstr ""
+msgstr "向量"
 
 #: ../../vector.rst:6
 msgid "A vector in Taichi can have two forms:"
-msgstr ""
+msgstr "在Taichi中，向量的表述有两种形式："
 
 #: ../../vector.rst:8
-msgid ""
-"as a temporary local variable. An ``n`` component vector consists of "
-"``n`` scalar values."
-msgstr ""
+msgid "as a temporary local variable. An ``n`` component vector consists of ``n`` scalar values."
+msgstr "作为临时局部变量，一个 ``n`` 分量的向量由标量 ``n`` 组成。"
 
 #: ../../vector.rst:9
-msgid ""
-"as an element of a global tensor. In this case, the tensor is an "
-"N-dimensional array of ``n`` component vectors."
-msgstr ""
+msgid "as an element of a global tensor. In this case, the tensor is an N-dimensional array of ``n`` component vectors."
+msgstr "作为全局张量(global tensor)的构成元素。比如，由向量 ``n`` 组成的N维序列构成了的一个全局张量。"
```

Comment:
Sorry about not mentioning this in [wiki](https://github.com/taichi-dev/taichi-docs-zh-cn/wiki/%E5%B8%B8%E7%94%A8%E4%B8%93%E6%9C%89%E8%AF%8D%E6%B1%87%E5%AF%B9%E7%85%A7%E8%A1%A8)! Adding this entry now.
```suggestion
msgstr "作为全局张量(global tensor)的构成元素。比如，由向量 ``n`` 组成的N维数组构成了的一个全局张量。"
```

Reply:
Lol.